Remove LGTM badge

alexfikl opened the pull request:
LGTM went EOL on December 15th, so this removes the badge.

@alejandrogallo The `coveralls` badge also points to something that still uses the TravisCI and hasn't been updated in a long time. Can you update that or should I remove it as well?

alejandrogallo commented:
I think we can safely remove it and at some point when we work more on test coverage we can put it back in with github ci

alexfikl commented:
> I think we can safely remove it and at some point when we work more on test coverage we can put it back in with github ci

Removed it! Also added a CodeQL badge to replace the LGTM one, since that's what we're using now.
